examples/deploy/go.work

```diff
@@ -0,0 +1,3 @@
+go 1.18
+
+use .
```

Comment:
the idea is that you remove the dependency to `github.com/dontpanicdao/caigo` in the go mod. I think adding `use ../..` should do it.